portal/server/utils/csv-utils.js

```diff
@@ -0,0 +1,137 @@
+const ExcelJS = require('exceljs');
```

Comment:
My mind melted slightly going through all the iterations of data processing I needed in this file, if anyone wants to suggest better variable/function naming etc. please do!